pyanaconda/modules/boss/boss.py

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/python3
+
+from pyanaconda.dbus import dbus_constants
+from pyanaconda.dbus.interface import dbus_interface
+from pyanaconda.dbus.typing import *  # pylint: disable=wildcard-import
```

Comment:
I don't see a reason for this import. Did I miss something?

Reply:
I guess it might not be needed right now.